types/semver/index.d.ts

```diff
@@ -1,118 +1,149 @@
-// Type definitions for semver 5.3
+// Type definitions for semver 5.4
 // Project: https://github.com/npm/node-semver
 // Definitions by: Bart van der Schoor <https://github.com/Bartvds>
+//                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/semver
 
 export const SEMVER_SPEC_VERSION: "2.0.0";
 
 export type ReleaseType = "major" | "premajor" | "minor" | "preminor" | "patch" | "prepatch" | "prerelease";
 
 /**
+ * Return an instance of SemVer, or null if it's not valid.
+ */
+export function parse(v: string | SemVer, loose?: boolean): SemVer | null;
+/**
  * Return the parsed version, or null if it's not valid.
  */
-export function valid(v: string, loose?: boolean): string;
+export function valid(v: string | SemVer, loose?: boolean): string | null;
 /**
  * Returns cleaned (removed leading/trailing whitespace, remove '=v' prefix) and parsed version, or null if version is invalid.
  */
-export function clean(version: string, loose?: boolean): string;
+export function clean(version: string, loose?: boolean): string | null;
 /**
  * Return the version incremented by the release type (major, minor, patch, or prerelease), or null if it's not valid.
  */
-export function inc(v: string, release: ReleaseType, loose?: boolean, identifier?: string): string;
+export function inc(v: string | SemVer, release: ReleaseType, loose?: boolean, identifier?: string): string | null;
```

Comment:
Done. It would be really helpful if you could merge this one in before you merge any other pull requests regarding `semver`.